### DISCUSSION TITLE
Is this project active?

### DISCUSSION THREAD

**Discussion by ghost** [NativeScript/NativeScript#10146] [Q&A] [answered]
Hi.

I want to help, but I want to know if this project is active. *Is this project active?*

**Comment by rigor789** [comment, marked as answer]
Hi, sure thing, it's active! You can look at the core repo for example, last commit was 5 days ago (at the time of writing this comment)

https://github.com/NativeScript/NativeScript/commits/main

**Reply by ghost** [reply]
Hi rigor789.

> Hi, sure thing, it's active! You can look at the core repo for example, last commit was 5 days ago (at the time of writing this comment)
> https://github.com/NativeScript/NativeScript/commits/main

thank you for feedback.
